pkg/hydrator/nexpose.go

```diff
@@ -2,38 +2,324 @@ package hydrator
 
 import (
 	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
 
 	"github.com/asecurityteam/nexpose-vuln-hydrator/pkg/domain"
 )
 
-type nexposeAssetVulnerability struct {
```

Comment:
question: why did you change this to an exported struct?

Reply:
`golint` complained about returning an unexported struct